api/HTMLSlotElement.json

```diff
@@ -238,10 +238,10 @@
               "version_added": false
             },
             "firefox": {
-              "version_added": true
+              "version_added": "66"
```

Comment:
As with the Chromium change, it would be great if changes that don't match the PR description are made in separate PR. Makes it easier to review and to understand the changes after the fact.

Reply:
I think you might have been looking at a commit that merges master — I never added version numbers, and this PR makes no changes to HTMLSlotElement.  :P

Reply:
hello, if i need some important matter .please point me and reply. i also learning from all of you. thanks